Ejercicio Práctico 12

Leidanys opened the pull request:
clase TestngSalesforce

Leidanys committed:
Ejercicio Práctico 12

seleniumcurso commented on TestngSalesforce.java:
```diff
@@ -0,0 +1,127 @@
+
+    @Test(priority = 1,groups = {"successTests"},enabled = true)
+    public void validateSalesforceLogoTest(){
+         driverChrome.get(SALEFORCE_URL);
```
esto va en el setup

seleniumcurso commented on TestngSalesforce.java:
```diff
@@ -0,0 +1,127 @@
+    @Test(priority = 1,groups = {"successTests"},enabled = true)
+    public void validateSalesforceLogoTest(){
+         driverChrome.get(SALEFORCE_URL);
+         driverChrome.manage().window().maximize();
```
esto va en el setup

seleniumcurso commented on TestngSalesforce.java:
```diff
@@ -0,0 +1,127 @@
+    public void validateSalesforceLogoTest(){
+         driverChrome.get(SALEFORCE_URL);
+         driverChrome.manage().window().maximize();
+         driverChrome.navigate().refresh();
```
esto va en el setup

seleniumcurso commented on TestngSalesforce.java:
```diff
@@ -0,0 +1,127 @@
+
+    @Test(priority = 4, groups = {"successTests"})
+    public void rememberMeIsSelected()throws Exception{
+       driverChrome.get("https://login.salesforce.com/?locale=eu");
```
esto va en el setup

seleniumcurso commented on TestngSalesforce.java:
```diff
@@ -0,0 +1,127 @@
+    @Test(priority = 4, groups = {"successTests"})
+    public void rememberMeIsSelected()throws Exception{
+       driverChrome.get("https://login.salesforce.com/?locale=eu");
+       driverChrome.manage().window().maximize();
```
esto va en el setup

seleniumcurso commented on TestngSalesforce.java:
```diff
@@ -0,0 +1,127 @@
+       driverChrome.navigate().refresh();
+
+       Thread.sleep(2000);
+       WebElement check= driverChrome.findElement(By.id("rememberUn"));
```
cambiar el nombre a checkboxRememberElement

seleniumcurso commented on TestngSalesforce.java:
```diff
@@ -0,0 +1,127 @@
+
+    @Test(priority = 2,enabled = false ,groups = {"fail test"})
+    public void FooterIsValid() throws Exception {
+        driverChrome.get(SALEFORCE_URL);
```
esto va en el setup

seleniumcurso commented on TestngSalesforce.java:
```diff
@@ -0,0 +1,127 @@
+    //Ejercicio GIT
+    @Test(priority = 3)
+    public void loginFailureTest(@Optional("test@test.com") String emailParameter, @Optional("123466") String passwordParameter) throws Exception{
+        driverChrome.get(SALEFORCE_URL);
```
esto va en el setup

seleniumcurso commented on TestngSalesforce.java:
```diff
@@ -0,0 +1,127 @@
+        Assert.assertEquals(driverChrome.getCurrentUrl(),SALEFORCE_URL, "No coincide la url");
+
+        Thread.sleep(2000);
+        WebElement logo= driverChrome.findElement(By.id("logo"));
```
cambiar el nombre a logoElement

seleniumcurso commented on TestngSalesforce.java:
```diff
@@ -0,0 +1,127 @@
+
+        try {
+            System.out.println("---->Limpiando cookies <-----------");
+            driverChrome.manage().deleteAllCookies();
```
utilizar solo un @AfterMethod

Leidanys committed:
Cambios:
- Se actualiza método setup: Se agrega el inicio de sesión del navegador. Maximizar y refrescar página.
- Se cambia el nombre de variable check por checkboxRememberElement. Método: rememberMeIsSelected.
- Se cambia nombre de variable logo por  logoElement. Método: loginFailureTest
- Se elimina método cleanCookies.

Leidanys commented:
Se realizó nuevo commit con los cambios  y se hizo el push.

seleniumcurso commented on TestngSalesforce.java:
```diff
@@ -0,0 +1,104 @@
+    @AfterMethod
+    private void closeDriver() throws Exception  {
+
+        try {
```
este try/catch no esta aportando al metodo. es redundante. Por favor quitarlo.

Leidanys committed:
Commit de la clase, se elimina el try catch del método closeDriver.

Leidanys commented:
close
